packages/components/src/templates/next/components/native/Table/Table.tsx

```diff
@@ -78,18 +78,18 @@ const Table = ({ caption, content }: TableProps) => {
             return (
               <tr
                 key={index}
-                className="*:first:border-divide-subtle *:first:border-t"
+                className="*:first:border-divide-subtle text-left *:first:border-t"
               >
                 {row.content.map((cell, cellIndex) => {
                   return (
                     <TableCellTag
                       key={cellIndex}
                       colSpan={cell.colSpan}
                       rowSpan={cell.rowSpan}
-                      className={`border-divide-subtle max-w-40 break-words border-b border-r px-4 py-3.5 align-top first:border-l last:max-w-full [&_li]:my-0 [&_li]:pl-1 [&_ol]:mt-0 [&_ol]:ps-5 [&_ol]:text-sm [&_ul]:mt-0 [&_ul]:ps-5 ${
+                      className={`border-divide-subtle max-w-40 break-words border-b border-r px-4 py-3.5 align-top first:border-l last:max-w-full [&_li]:my-0 [&_li]:pl-1 [&_ol]:mt-0 [&_ol]:ps-5 [&_ol]:text-sm [&_p]:text-sm [&_ul]:mt-0 [&_ul]:ps-5 ${
                         cell.type === "tableHeader"
                           ? "bg-utility-neutral"
-                          : "bg-utility-neutral-subtle [&_p]:text-sm"
+                          : "bg-utility-neutral-subtle"
```

Comment:
actually just a point here - we now rely exclusively on the background to differentiate between header-row and cell. 

going off notion's one, they seem to have the header row having slightly heavier `font-weight` (and is also what i'd expect off my web browsing habits). wondering how much extra effort is required for this to conform w/ user's expectations
![image](https://github.com/opengovsg/isomer/assets/44049504/5e4c6ba2-cc5f-4435-9c40-22f6facd7008)

Reply:
using font-weight to differentiate header as well! this screenshot is from this branch:

<img width="416" alt="image" src="https://github.com/opengovsg/isomer/assets/139780851/4e33b7da-67f5-40ec-9572-942b7566aa92">